bot/engine/src/test/kotlin/connector/ConnectorQueueTest.kt

```diff
@@ -0,0 +1,93 @@
+package ai.tock.bot.connector
+
+import ai.tock.bot.engine.action.Action
+import ai.tock.bot.engine.user.PlayerId
+import ai.tock.shared.Executor
+import ai.tock.shared.vertx.blocking
+import ai.tock.shared.vertx.vertx
+import io.mockk.Ordering
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.spyk
+import io.mockk.verify
+import org.junit.jupiter.api.Test
+import java.time.Duration
+import java.util.concurrent.Callable
+
+internal class ConnectorQueueTest {
+
+    private object executor : Executor {
+        override fun executeBlocking(delay: Duration, runnable: () -> Unit) {
+            vertx.blocking<Unit>({
+                runnable.invoke()
+            }, {})
+        }
+
+        override fun executeBlocking(runnable: () -> Unit) {
+            TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
```

Comment:
J'aurais plutôt mis un `throw NotImplemented`, ce TODO ne sera jamais fait j'imagine

Reply:
ok > https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-not-implemented-error/index.html